Gemfile

```diff
@@ -93,7 +93,7 @@ gem 'bootsnap', require: false
 gem 'geocoder'
 gem 'gmaps4rails'
 gem 'mimemagic', '> 0.3.5'
-gem 'paper_trail', '~> 12.1'
+gem 'paper_trail', '~> 15.1'
```

Comment:
Shall we remove this restriction so that Depandabot will suggest future updates?